### PR TITLE
dztech rgb pcb v2 support

### PR DESCRIPTION
**Pull request by moyi4681**
dztech rgb pcb v2 support

### CODE REVIEW

File: keyboards/dztech/dz60rgb/v2/rules.mk [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb/v2/rules.mk]: Lines 3-40 should be replaced with:

```Makefile
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = atmel-dfu
```

File: keyboards/dztech/dz60rgb_ansi/v1/rules.mk [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
 CONSOLE_ENABLE = no                # Console for debug
 COMMAND_ENABLE = no                # Commands for debug and configuration
 #SLEEP_LED_ENABLE = yes            # Breathing sleep LED during USB suspend
-NKRO_ENABLE = no                 # USB Nkey Rollover
+NKRO_ENABLE = yes                  # USB Nkey Rollover
 AUDIO_ENABLE = no
 RGB_MATRIX_ENABLE = IS31FL3733     # Use RGB matrix
 NO_USB_STARTUP_CHECK = no          # Disable initialization only when usb is plugged in
-
-LAYOUTS = 60_ansi
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb_ansi/v1/rules.mk]: Is there a particular reason for this removal?

File: keyboards/dztech/dz60rgb_ansi/v2/rules.mk [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb_ansi/v2/rules.mk]: Same as above.

```Makefile
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = atmel-dfu
```

File: keyboards/dztech/dz60rgb_wkl/v2/rules.mk [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb_wkl/v2/rules.mk]:
```
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = atmel-dfu
```

File: keyboards/dztech/dz60rgb/v2/rules.mk (hunk 1 above)
Comment (drashna) [reply keyboards/dztech/dz60rgb/v2/rules.mk]: Specifically, most of this set bu default, so it only needs to be specified, if you're setting it to value that is something other than the default.  

This makes it easier to use/manage.

File: keyboards/dztech/dz60rgb_ansi/v1/rules.mk (hunk 2 above)
Comment (drashna) [reply keyboards/dztech/dz60rgb_ansi/v1/rules.mk]: If this is specifically due to community keymaps erroring out, that's okay.  Not all of them are tested on every keyboard. But as long as the default keymap compiles, then it should be fine.

File: keyboards/dztech/dz60rgb/v2/rules.mk [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+BACKLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb/v2/rules.mk]:
```suggestion

BACKLIGHT_ENABLE = no
```

File: keyboards/dztech/dz60rgb/v2/rules.mk [hunk 6]
```diff
@@ -0,0 +1,21 @@
+MCU = atmega32u4
+# Bootloader selection
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb/v2/rules.mk]:
```suggestion

# Bootloader selection
```

File: keyboards/dztech/dz60rgb_ansi/v2/rules.mk [hunk 7]
```diff
@@ -0,0 +1,21 @@
+MCU = atmega32u4
+# Bootloader selection
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_ansi/v2/rules.mk]:
```suggestion

# Bootloader selection
```

File: keyboards/dztech/dz60rgb_ansi/v2/rules.mk [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+BACKLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_ansi/v2/rules.mk]:
```suggestion

BACKLIGHT_ENABLE = no
```

File: keyboards/dztech/dz60rgb_wkl/v2/rules.mk [hunk 9]
```diff
@@ -0,0 +1,21 @@
+MCU = atmega32u4
+# Bootloader selection
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_wkl/v2/rules.mk]:
```suggestion

# Bootloader selection
```

File: keyboards/dztech/dz60rgb_wkl/v2/rules.mk [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+BACKLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_wkl/v2/rules.mk]:
```suggestion

BACKLIGHT_ENABLE = no
```

**Pull request merged**